### ISSUE TITLE
homebrew package is still 0.1.4, not 0.1.6

### ISSUE THREAD

**Issue by beckettsean** [influxdb/telegraf#146]
```
sean@catalyst-sean:~$ brew update
Already up-to-date.
sean@catalyst-sean:~$ brew install telegraf
==> Downloading https://homebrew.bintray.com/bottles/telegraf-0.1.4.yosemite.bottle.tar.gz
```

**Comment by sparrc** [comment]
I'm going to make a 0.1.7 release that supports kafka output and update the homebrew formula to that today

**Comment by sparrc** [comment]
@beckettsean this should be fixed now, homebrew should install 0.1.7 now (also linked on the README for linux)

**Comment by beckettsean** [comment]
confirming brew now gives me 0.1.7
